COLLABORATOR_GUIDE.md

```diff
@@ -292,25 +292,23 @@ metadata. Raise a Pull Request like any other change.
 
 ### Introducing New Modules
 
-Semver-minor commits that introduce new core modules should be treated with
-extra care.
+Treat commits that introduce new core modules with extra care.
```

Comment:
Is it mentioned elsewhere that introducing a new core module is semver-minor?

Reply:
Not as far as I know. If it's desirable to have it here, I can add a sentence. I'd prefer to leave it out as it's self-evident and low-value (as someone else will certainly add the label if the author does not) and this doc is plenty long as it is. But I don't have a *strong* preference, so I'll yield if someone thinks it needs to be in there.

Reply:
I don't really have a strong preference either. I guess if we do mention it it can go in the bullet list below (probably next to "Label with the `tsc-review` label.").

Reply:
By definition it adds functionality, ergo, `semver-minor`.